lib/travis/api/v3/services/cron/create.rb

```diff
@@ -10,7 +10,7 @@ def run!
       raise Error.new('Invalid value for interval. Interval must be "daily", "weekly" or "monthly"!', status: 422) unless ["daily", "weekly", "monthly"].include?(params["interval"])
       access_control.permissions(repository).create_cron!
       access_control.permissions(branch.cron).delete! if branch.cron
-      query.create(branch, params["interval"], params["disable_by_build"] ? params["disable_by_build"] : false)
+      query.create(branch, params["interval"], params["run_only_when_new_commit"] ? params["run_only_when_new_commit"] : false)
```

Comment:
I don't think you need this ternary. You should be able to use the value of params["run_only_when_new_commit"] since this logic implies it is either true, false or nil.